ab_testing_tool/settings/test_settings.py

```diff
@@ -7,3 +7,12 @@
 if not ENV_SETTINGS.get('enable_test_logging'):
     for logger in LOGGING['loggers']:
         LOGGING['loggers'][logger]['handlers'] = ['null']
+
+# make tests faster
+DATABASES = {
```

Comment:
I could also configure my postgres user to have privileges to create dbs, but seems like it's simpler/faster to have the test db use sqlite until we start doing something supported by postgres and not in sqlite.